charts/trino/values.yaml

```diff
@@ -146,3 +146,19 @@ worker:
     # requests:
     #   cpu: 100m
     #   memory: 128Mi
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: trino.example.com
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
```

Comment:
Just comment this out to avoid confusion around default values.

```suggestion
#  hosts:
#    - host: trino.example.com
#      paths:
#        - path: /
#          pathType: ImplementationSpecific
```